keybot/platform_darwin.go

```diff
@@ -27,6 +27,10 @@ func buildAndroidCommand() slackbot.ExecCommand {
 	return slackbot.NewExecCommand("/bin/launchctl", []string{"start", "keybase.android.release"}, false, "Perform an alpha build")
 }
 
+func buildIOSCommand() slackbot.ExecCommand {
+	return slackbot.NewExecCommand("/bin/launchctl", []string{"start", "keybase.ios.release"}, false, "Perform an testflight build")
```

Comment:
i wonder if we should be calling these .release or .testflight   ... same with android above?